3_summarize/src/annual_thermal_metrics.R

```diff
@@ -1,13 +1,14 @@
 
-calculate_annual_metrics_per_lake <- function(site_id, site_file, ice_file, morphometry) {
+calculate_annual_metrics_per_lake <- function(out_file, site_id, site_file, ice_file, morphometry) {
   
   start_tm <- Sys.time()
```

Comment:
I like that idea! Gives people the option. This was originally just for testing purposes to see how long individual lakes were taking. I'll add this in!

Reply:
Cool! I would suggest using `verbose = FALSE` as the default argument, so that you override that when you specify `verbose = I(TRUE)` in the inputs.

Reply:
Hmmm so I just tried implementing this but it doesn't seem to print those messages top-level anyways. If I run the individual target from the task table, I see the message if I have `verbose = TRUE` but if I run the target that does `do_annual_metrics_multi_lake` which uses `loop_tasks` to build the targets, then I don't get the message.

```r
> scmake("3_summarize/tmp/nhdhr_80996367_annual_thermal_metrics.rds", "3_summarize_pb0_metric_tasks.yml")
Starting build at 2020-11-18 09:19:09
<  MAKE > 3_summarize/tmp/nhdhr_80996367_annual_thermal_metrics.rds
[    OK ] sb_dl_date
[    OK ] 1_fetch/out/config.json
[    OK ] morphometry
[    OK ] nhdhr_80996367_morphometry
[ BUILD ] 3_summarize/tmp/nhdhr_80996367_annual_thermal_metrics.rds |  calculate_annual_metrics_per_lake(out_file = "3_summar...
Completed annual metrics for nhdhr_80996367 in 0.1 min
Finished build at 2020-11-18 09:19:16
Build completed in 0.10 minutes
```

When I ran `scmake("3_summarize/out/annual_metrics_pb0.csv")`, I didn't get the message (even though it did rebuild because I deleted the `.rds` file:

```r
> scmake("3_summarize/out/annual_metrics_pb0.csv")
Starting build at 2020-11-18 09:19:42
<  MAKE > 3_summarize/out/annual_metrics_pb0.csv
[    OK ] sb_dl_date
[    OK ] toha_sbid
[    OK ] clarity_ice_sbid
[    OK ] 1_fetch/out/lake_metadata.csv
[    OK ] 1_fetch/out/irradiance_downloaded.yml
[    OK ] 1_fetch/out/pb0_temp_pred_downloaded.yml
[    OK ] 1_fetch/out/clarity_downloaded.yml
[    OK ] 1_fetch/out/iceflags_downloaded.yml
[    OK ] sb_group_info
[    OK ] sb_group_ids
[    OK ] process_grp_tasks
[    OK ] 2_process_grp_tasks.yml
[    OK ] 2_process/out/2_process_grp_tasks_completed.yml
[    OK ] 2_process/out/iceflags_unzipped.yml
[    OK ] ice_files
[    OK ] pb0_site_files
[    OK ] task_file
[ BUILD ] 3_summarize/out/annual_metrics_pb0.csv           |  do_annual_metrics_multi_lake(final_target = "3_summarize/out/an...
v0.0.17: tickquote_combinee_objects is deprecated and will effectively be always TRUE in future versions
Run all tasks and finalizers with:
3_summarize/out/annual_metrics_pb0.csv:
  command: scmake(I('annual_metrics_pb0.csv_promise'), remake_file='D:/LRCData/R/lake-temperature-out/3_summarize_pb0_metric_tasks.yml')
### Starting loop attempt 1 of 1 with 1 of 1 tasks remaining:                                                                        

### Final check for completeness of all targets
Starting build at 2020-11-18 09:20:14
<  MAKE > 3_summarize_pb0_metric_tasks
[    OK ] sb_dl_date
[    OK ] 1_fetch/out/config.json
[    OK ] morphometry
[    OK ] nhdhr_80996367_morphometry
[    OK ] 3_summarize/tmp/nhdhr_80996367_annual_thermal_metrics.rds
[    OK ] annual_metrics_pb0.csv_promise
[ ----- ] 3_summarize_pb0_metric_tasks
Finished build at 2020-11-18 09:20:14
Build completed in 0.00 minutes
Finished build at 2020-11-18 09:20:15
Build completed in 0.56 minutes
```

Reply:
Is that the expected behavior?

Reply:
I'm not quite sure. I think `loop_tasks` does some kind of message suppression, but I didn't expect it to hush this particular message. `loop_tasks` has its own `verbose` argument, which is how you control whether to print the progress bar. You could switch that off and see if it works. Or ignore since this doesn't really seem to be much of a problem in reality, right?

Reply:
Hmm that didn't seem to make a difference. I will just commit what I've got and let it go :)